draft-ietf-tls-esni.md

```diff
@@ -425,16 +412,27 @@ The client-facing server computes ClientHelloInner by reversing this process.
 It scans EncodedClientHelloInner for an "outer_extensions" extension. If there
 is none, the ClientHelloInner is the EncodedClientHelloInner. Otherwise, it
 replaces the extension with the corresponding sequence of extensions in
-the ClientHelloOuter. If any referenced extensions are missing, the server
-MUST abort the connection with an "illegal_parameter" alert.
-The server then computes the digest of the reconstructed
-ClientHelloInner. If the digest does not equal OuterExtensions.inner_digest,
-then the server MUST abort the connection with a "decrypt_error" alert.
+the ClientHelloOuter. If any referenced extensions are missing or if
+"encrypted_client_hello" appears in the list, the server MUST abort the
+connection with an "illegal_parameter" alert.
 
 The "outer_extensions" extension is only used for compressing the
 ClientHelloInner. It MUST NOT be sent in either ClientHelloOuter or
 ClientHelloInner.
 
+## Authenticating the Outer ClientHello {#authenticating-outer-clienthello}
```

Comment:
Suggestion: s/Outer ClientHello/ClientHelloOuter.

Reply:
(Will deal with uploading a new revision after base PRs go through, since it's all a mess.)

Works for me, though it's unclear to me how much we like sticking identifiers in section titles, which is why I used the words instead. E.g. RFC8446's section titles say things like "Supported Versions" and this document had "Incorporating Outer Extensions" rather than "outer_extensions". Although, admittedly, "ClientHello" is itself an identifier, but I figured it was a more established one (elsewhere we have "Sending an encrypted ClientHello").

¯\\\_(ツ)_/¯

Reply:
Oh, and we have "Outer ClientHello" as a section title down in Security Considerations, though I suppose that too is my fault. :-)

Reply:
Done.